### ISSUE TITLE
Bug:  impossible address

### ISSUE THREAD

**Issue by Hyper-Tex** [Tau-ri-Dev/JSGMod-1.12.2#171]
I'm not sure if this has happened before, and I'm pretty sure it's just bad luck on my part, but when messing around with the mod, I got this impossible address. the point of origin is the second symbol
![Screenshot (100)](https://user-images.githubusercontent.com/17879365/177328265-463f96b0-bae8-47f7-b41e-8b9b3d828c09.png)
it's on the latest version of the mod available on curseforge aunis-1.12.2-4.10.1.0-client.jar

I'm not sure there's anything I can add, I do play with a bunch of other mods, but this looks like an issue with JSG itself

**Comment by MikiCZSK** [comment]
I second this Happend to me too. I also have the latest version af of 6/7/2022. Probably system that excludes points of orgin from adresses isnt working properly. 
![2022-07-06_11 21 13](https://user-images.githubusercontent.com/65219648/177517205-1a1d2411-ff21-436f-abec-b788875be3f5.png)

**Comment by MikiCZSK** [comment]
This effectively makes any Pegasus to MW gate impossible. Tried it with OC and it wasnt working.

**Comment by MineDragonCZ** [comment]
> I'm not sure if this has happened before, and I'm pretty sure it's just bad luck on my part, but when messing around with the mod, I got this impossible address. the point of origin is the second symbol ![Screenshot (100)](https://user-images.githubusercontent.com/17879365/177328265-463f96b0-bae8-47f7-b41e-8b9b3d828c09.png) it's on the latest version of the mod available on curseforge aunis-1.12.2-4.10.1.0-client.jar
> 
> I'm not sure there's anything I can add, I do play with a bunch of other mods, but this looks like an issue with JSG itself

Also is it only the texture?

**Comment by Hyper-Tex** [comment]
Ah, unfortunately another mod destroyed the problem gate a while back, and I don't remember exactly where I built it to recreate it the issue, but I do know that it wasn't a texture issue. It was the first thing I tested by uninstalling every texture pack I had, along with breaking and rebuilding the gate in the same spot. It was in the Pegasus glyph UI itself. It obviously has happened before to other people, as proven by MikiCZSK. I'm thinking it's a rare glitch, maybe rare enough to not bother with.
